### DISCUSSION TITLE
How can we accomodate different ways to form strings in stratigraphy?

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#1125] [Q&A]
Suggestion from IT: 
 
For the first part of this we've had a think and our suggested solution is to add an "Automatic?" Yes/No to determine if a name is generated automatically using the existing rules or not. We would then default it to "Yes" for fossil inverts/paleobotany groups and no for fossil vertebrates groups. In the case of a "No" the user is responsible for entering the data as it needs to be formed. This is similar to the way both parties and taxonomy work.
 
![AllSumTab_2024](https://github.com/fieldmuseum/EMu-Documentation/assets/702709/8bb47318-8995-4019-b135-f7afe708b1a1)
 
For the second part a verbatim/informal field on the AllOthTab is an option when there is no official/correctly formed/agreed name. 
 
![AllOthTab](https://github.com/fieldmuseum/EMu-Documentation/assets/702709/7752435c-2857-47a7-bf25-a8023ebd5873)
 
To your point about uncertainty around the strat for the specimen itself our recommendation is to add in the standard qualifier fields as are present on the identification tab in the catalogue. This will handle the usual possible/probable etc. values. In the meantime there is currently a "Comment" field on the stratigraphy tab in catalogue that should be used for non standard information about the stratigraphy.
 
![AllStrTab](https://github.com/fieldmuseum/EMu-Documentation/assets/702709/5a3a7c44-de23-4f63-a234-a6f5dae6fc1f)
 
See if that addresses the issues you are bumping into.
 
S

**Comment by rondlg** [comment]
[Paul Mayer](https://fieldmuseum.zendesk.com/agent/users/144472418)
13 minutes ago
To:IT HelpdeskShow more
The first and the last ideas seem great to me.

The second idea should work might be better if this was on the lithostratigraphic page. of if there was an informal term on the litho page.

I share this with Geology collections folks and it seemed to go over fine. They will look at it for a bit and let me know what they think and I will let you know as soon as I hear.

Paul

**Comment by rondlg** [comment]
@cmaves3 this is where you can document the conversations around this going forward. I would like to submit stratigraphy structural changes to Axiell next week for a quote.

**Comment by rondlg** [comment]
William Simpson | William Simpson | 16:22 (1 minute ago) |   | Reply
William Simpson
to Paul, Chris, Adrienne, James, Michael, me | to Paul, Chris, Adrienne, James, Michael, me
to Paul, Chris, Adrienne, James, Michael, me

I like the first solution; a yes/no toggle to allow us to enter formation names the way we choose seems perfect.

The second solution is less elegant given that (I think?) the various strata (both formal and informal) will not be in same window.  To me, this is one of EMu's weaknesses; separating related data across multiple windows.  But maybe I'm not understanding how this will work yet.

The third solution, to add a qualifier field (to hold a "?", for example) seems good.  I assume there would be a qualifier field for each class of strata (Group, Formation, Member, and any informal strata?

Thanks for addressing these concerns, Sharon, Chris and Paul!

**Reply by cmaves3** [reply]
> Reply to William Simpson
> The third solution, to add a qualifier field (to hold a "?", for example) seems good.  I assume there would be a qualifier field for each class of strata (Group, Formation, Member, and any informal strata?

This field would be in the Catalogue module (Catalogue record of the specimen) so the qualifier would be for the entire Stratigraphy ID that is attached to it. Could we possibly use Qualifier Rank as also used on Identification tab?

**Comment by rondlg** [comment]
This is what the Lithostrat tab looks like with the informal name field on it.
![AllLitTab](https://github.com/fieldmuseum/EMu-Documentation/assets/702709/5267ff6b-d962-4563-aaf4-aa52f93040c2)

**Comment by cmaves3** [comment]
From Bill,

For the Menu list, could we reorder the different stratigraphic fields like this:

Chronostratigraphy

Lithostratigraphy

Biostratigraphy

Human Cultural Phase

Absolute dating 

Relative Dating

Other Stratigraphy
The first two (bolded) will almost always be used, the others much less often. This will make entering data smoother, more intelligible, and thus, more efficient.

Thanks!

Best,
Bill

**Reply by rondlg** [reply]
Hi, which menu list are you referring to?

**Reply by cmaves3** [reply]
This is for the Summary tab

**Comment by rondlg** [comment]
Add Rank to the qualifier logic to the catalogue module on the stratigraphy tab, that works in the same way as it does on the identification tab.

**Comment by fmjjones** [comment]
Additional request to remove "bed" from the Bed field in Lithostratigraphy Zendesk Ticket [#74368](https://fieldmuseum.zendesk.com/agent/tickets/74368)
